examples/experimental/redis.js

```diff
@@ -1,87 +1,83 @@
-import { check } from "k6";
-import http from "k6/http";
-import redis from "k6/experimental/redis";
-import exec from "k6/execution";
-import { textSummary } from "https://jslib.k6.io/k6-summary/0.0.1/index.js";
+import { check } from 'k6';
+import http from 'k6/http';
+import redis from 'k6/experimental/redis';
+import exec from 'k6/execution';
+import { textSummary } from 'https://jslib.k6.io/k6-summary/0.0.2/index.js';
+
 export const options = {
 	scenarios: {
 		redisPerformance: {
-			executor: "shared-iterations",
+			executor: 'shared-iterations',
 			vus: 10,
 			iterations: 200,
-			exec: "measureRedisPerformance",
+			exec: 'measureRedisPerformance',
 		},
 		usingRedisData: {
-			executor: "shared-iterations",
+			executor: 'shared-iterations',
 			vus: 10,
 			iterations: 200,
-			exec: "measureUsingRedisData",
+			exec: 'measureUsingRedisData',
 		},
 	},
 };
-// Get the redis instance(s) address and password from the environment
-const redis_addrs = __ENV.REDIS_ADDRS || "";
-const redis_password = __ENV.REDIS_PASSWORD || "";
+
 // Instantiate a new redis client
-const redisClient = new redis.Client({
-	addrs: redis_addrs.split(",") || new Array("localhost:6379"), // in the form of 'host:port', separated by commas
-	password: redis_password,
-});
-// Prepare an array of crocodile ids for later use
+const redisClient = new redis.Client(`redis://localhost:6379`);
+
+// Prepare an array of rating ids for later use
 // in the context of the measureUsingRedisData function.
-const crocodileIDs = new Array(0, 1, 2, 3, 4, 5, 6, 7, 8, 9);
-export function measureRedisPerformance() {
+const ratingIDs = new Array(0, 1, 2, 3, 4, 5, 6, 7, 8, 9);
+
+export async function measureRedisPerformance() {
 	// VUs are executed in a parallel fashion,
 	// thus, to ensure that parallel VUs are not
 	// modifying the same key at the same time,
 	// we use keys indexed by the VU id.
 	const key = `foo-${exec.vu.idInTest}`;
-	redisClient
-		.set(`foo-${exec.vu.idInTest}`, 1)
-		.then(() => redisClient.get(`foo-${exec.vu.idInTest}`))
-		.then((value) => redisClient.incrBy(`foo-${exec.vu.idInTest}`, value))
-		.then((_) => redisClient.del(`foo-${exec.vu.idInTest}`))
-		.then((_) => redisClient.exists(`foo-${exec.vu.idInTest}`))
```

Comment:
Do we no longer need to use `${exec.vu.idInTest}`?

Reply:
Same as above!